userspace/libsinsp/test/filterchecks/evt.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (C) 2023 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <test/helpers/threads_helpers.h>
+
+TEST_F(sinsp_with_test_input, EVT_FILTER_is_open_create)
+{
+	add_default_init_thread();
+
+	open_inspector();
+
+	std::string path = "/home/file.txt";
+	int fd = 3;
```

Comment:
```suggestion
	int64_t fd = 3;
```
This should solve the last failing tests! :tada:

Reply:
Great catch Andrea!